cvat-core/src/frames.ts

```diff
@@ -728,7 +744,9 @@ export async function getContextImage(jobID: number, frame: number): Promise<Rec
     const meta = await frameData.getMeta();
     const requestId = frame;
     const { jobStartFrame } = frameData;
-    const { related_files: relatedFiles } = meta.frames[frame - jobStartFrame];
+    const dataFrameNumber = meta.getDataFrameNumber(frame - jobStartFrame);
+    const frameIndex = meta.getFrameIndex(dataFrameNumber);
+    const { related_files: relatedFiles } = meta.frames[frameIndex];
```

Comment:
Seems like a fix not related to the main PR, I will propose to create a separate pull request

Reply:
Ok, https://github.com/cvat-ai/cvat/pull/9162